submit.js

```diff
@@ -0,0 +1,276 @@
+        level: 3 // 지도의 확대 레벨
+    };
+// 지도를 표시할 div와  지도 옵션으로  지도를 생성합니다
+var map = new kakao.maps.Map(mapContainer, mapOption);
+
+// 장소 검색 객체를 생성합니다
+const placeshow = new kakao.maps.services.Places();
+
+// 검색 결과 목록이나 마커를 클릭했을 때 장소명을 표출할 인포윈도우를 생성합니다
+var infowindow = new kakao.maps.InfoWindow({ zIndex: 1 });
+
+// 키워드 검색을 요청하는 함수입니다
+function searchPlaces() {
+    let keyword = document.getElementById('keyword').value;
+
+    // 장소검색 객체를 통해 키워드로 장소검색을 요청합니다
+    placeshow.keywordSearch(keyword, placesSearchCB);
+}
+
+// 장소검색이 완료됐을 때 호출되는 콜백함수 입니다
+function placesSearchCB(data, status, pagination) {
+    if (status === kakao.maps.services.Status.OK) {
+
+        // 정상적으로 검색이 완료됐으면
+        // 검색 목록과 마커를 표출합니다
+        displayPlaces(data);
+
+        // 페이지 번호를 표출합니다
+        displayPagination(pagination);
+
+    } else if (status === kakao.maps.services.Status.ZERO_RESULT) {
+
+        alert('검색 결과가 존재하지 않습니다.');
+        return;
+
+    } else if (status === kakao.maps.services.Status.ERROR) {
+
+        alert('검색 결과 중 오류가 발생했습니다.');
+        return;
+    }
+}
+
+// 검색 결과 목록과 마커를 표출하는 함수입니다
+function displayPlaces(places) {
+
+    var listEl = document.getElementById('placesList');
+    var menuEl = document.getElementById('menu_wrap');
+    var fragment = document.createDocumentFragment();
+    var bounds = new kakao.maps.LatLngBounds();
+    var listStr = '';
+
+    // 검색 결과 목록에 추가된 항목들을 제거합니다
+    removeAllChildNods(listEl);
+
+    // 지도에 표시되고 있는 마커를 제거합니다
+    removeMarker();
+
+    for (var i = 0; i < places.length; i++) {
+
+        // 마커를 생성하고 지도에 표시합니다
+        var placePosition = new kakao.maps.LatLng(places[i].y, places[i].x),
+            marker = addMarker(placePosition, i),
+            itemEl = getListItem(i, places[i]); // 검색 결과 항목 Element를 생성합니다
+        storeX = places[i].x;
+        storeY = places[i].y;
+        roadName = places[i].road_address_name;
+        storeNumber = places[i].phone;
+
+        // 검색된 장소 위치를 기준으로 지도 범위를 재설정하기위해
+        // LatLngBounds 객체에 좌표를 추가합니다
+        bounds.extend(placePosition);
+
+        // 마커와 검색결과 항목에 mouseover 했을때
+        // 해당 장소에 인포윈도우에 장소명을 표시합니다
+        // mouseout 했을 때는 인포윈도우를 닫습니다
+        (function (marker, title, storeNumber, roadName, storeX, storeY) {
+            kakao.maps.event.addListener(marker, 'mouseover', function () {
+                displayInfowindow(marker, title);
+            });
+
+            kakao.maps.event.addListener(marker, 'mouseout', function () {
+                infowindow.close();
+            });
+
+            kakao.maps.event.addListener(marker, 'click', function () {
+                console.log(roadName);
+                //displayMap();
+                addToDb(title, storeX, storeY, storeNumber, roadName);
+            });
+
+            itemEl.onmouseover = function () {
+                displayInfowindow(marker, title);
+            };
+
+            itemEl.onmouseout = function () {
+                infowindow.close();
+            };
```

Comment:
```suggestion
            itemEl.onmouseenter = function () {
                displayInfowindow(marker, title);
            };

            itemEl.onmouseleave = function () {
                infowindow.close();
            };
```

mouseover, mouseout은 자식요소들에게서도 이벤트가 발생합니다.
그래서 핸들러가 요소 안에서 자식요소들간에 마우스커서가 이동할때마다 실행될겁니다.
의도한게 아니라면
mouseenter, mouseleave를 사용하여 부모 요소에서만 이벤트를 발생시켜서 핸들러를 중복 실행하지 않게 하면 좋겠네요.